src/core/root.js

```diff
@@ -20,6 +20,8 @@ import { type RootState, defaultLogSettings, reducer } from './root-reducer.js'
 
 let allContexts: EdgeContext[] = []
 
+const ACCEPTED_SERVER_DOMAINS = ['edge.app', 'edgetest.app', 'localhost']
```

Comment:
This is not the approach we agreed on back in the day - we were going to have the server list hard-coded internally, and then accept an enum like `'localhost' | 'test' | 'production'` to select among the limited set of options. This would eventually allow us to hook into the info server as well, since we have a set of known keys to trigger off of.

In either case, this all seems out-of-scope for this PR, unless you want to prioritize this right away as a P1.

Reply:
I do like the enum for the sake of triggering off the info server. But for development/testing, the open string seems more flexible.

How about `authServer === undefined` implies we can grab off the info server. But if you pass a string, we validate and use that? Seems to cover all use cases.